docs/support-matrix.md

```diff
@@ -0,0 +1,51 @@
+# Compatible Storage Providers
+
+Ark is compatible with a variety of storage providers for backup data, as well as for volume snapshots using either a plugin for the volume provider, or [restic][4] for filesystem level backups.
+
+As of Ark v0.6.0, Ark provides a plugin system that allows anyone to add compatibility for additional backup and volume storage platforms without modifying the Ark codebase.
+
+## Backup Storage Providers
+
+| Provider                  | Owner    | Contact                         |
+|---------------------------|----------|---------------------------------|
+| [AWS S3][2]               | Ark Team | [Slack][10], [GitHub Issue][11] |
+| [Azure Blob Storage][3]   | Ark Team |                                 |
+| [Google Cloud Storage][4] | Ark Team |                                 |
+
+## S3-Compatible Backup Storage Providers
+
+Ark interfaces with the S3 API via [Amazon's Go SDK][12]. Some third party object storage providers use the S3 API, and users have reported the following will work using Ark's S3 support.
+
+Note that these providers are not regularly tested by the Ark team.
+
+ * [IBM Cloud][5]
+ * [Minio][9]
+ * Ceph RADIOS v12.2.7
+
+## Volume Snapshot Providers
+| Provider                         | Owner           | Contact                         |
+|----------------------------------|-----------------|---------------------------------|
+| [AWS EBS][2]                     | Ark Team        | [Slack][10], [GitHub Issue][11] |
+| [Azure Managed Disks][3]         | Ark Team        |                                 |
+| [Google Compute Engine Disks][4] | Ark Team        |                                 |
+| [Portworx][6]                    | PortWorx        |                                 |
+| [DigitalOcean][7]                | StackPointCloud |                                 |
+
+### Adding a new plugin
+
+If you would like to write a plugin for a new backup or volume storage system, take a look at the [example repo][8].
+
+Once you've published your plugin, open a PR adding the relevant information to the tables above.
```

Comment:
s/Once you've published/After you publish
s/adding the relevant information to the tables above./that adds your plugin to the appropriate list.